isis/tests/TestCsmModel.cpp

```diff
@@ -484,4 +484,147 @@ std::vector<double> TestCsmModel::getCrossCovarianceMatrix(
   return covariance;
 }
 
+// csm::RasterGM methods
+csm::ImageCoord TestCsmModel::groundToImage(const csm::EcefCoord& groundPt,
```

Comment:
I'm not really sure where the testcsmmodel is being used

Reply:
This gets used in the csminit tests. They use TestCsmPlugin which creates TestCsmModel and AlternativeTestCsmModel